environments/manager/images.yml

```diff
@@ -22,6 +22,10 @@ phpmyadmin_image: "{{ docker_registry }}/phpmyadmin/phpmyadmin:{{ phpmyadmin_tag
 postgres_tag: "15.3-alpine"
 postgres_image: "{{ docker_registry }}/library/postgres:{{ postgres_tag }}"
 
+postgres_upgrade_tag: "14-to-15"
+postgres_upgrade_image: "{{ docker_registry }}/tianon/postgres-upgrade:{{ postgres_upgrade_tag }}"
```

Comment:
not sure where this comes from? do we need it?

Reply:
Is in preparation to be able to upgrade Postgres properly in the future on the manager.

Reply:
hmm, o.k., but why does the bot add it? or is that in reaction to a different change?

Reply:
The bot synchronizes the environments/manager/images.yml. If a new image is added in cfg-generics or in release, it is added in the testbed via the bot.